lib/buildkite/test_collector/uploader.rb

```diff
@@ -51,5 +51,14 @@ def self.upload(data)
         end
       end
     end
+
+    def self.summary
+      return false unless Buildkite::TestCollector.api_token
+
+      http = Buildkite::TestCollector::HTTPClient.new("#{Buildkite::TestCollector.url}/summary")
```

Comment:
@niceking this has been updated here: [PIE-2047](https://github.com/buildkite/buildkite/pull/14288/files#diff-f1b550c16dfbb29caae2f9eebd8c9fbda11129ab77dde799a2097175a91b700dR91 ). I'm using the url `.../uploads/summary` as we send the uploads url through to the test collector, so this means if we created a new url for summary we will also need to send this through, maybe I am just being lazy here though and it is better to send through another unique url?